docs/dev/api/storage.md

```diff
@@ -405,6 +405,220 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 ```
 </details>
 
+---
+### Get App Storage Group Settings
+
+<details><summary><span className="api get">GET</span> <code>/v1/storage/groups/{group_id}/settings</code></summary>
+<p/>
+
+Returns the settings of the requested app group (apps containing multiple files).
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>group_id</code></td>
+     <td><p><small>| PATH | REQUIRED | INTEGER |</small></p><p>ID of the group of which settings need to be fetched.</p></td>
+    </tr>
+  </tbody>
+
+</table>
+
+
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
+{label: 'Europe', value: 'eu'},
+]}>
+
+<TabItem value="us">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.us-west-1.saucelabs.com/v1/storage/groups/2175303/settings' | json_pp
+```
+
+</TabItem>
+<TabItem value="eu">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" \
+--request GET 'https://api.eu-central-1.saucelabs.com/v1/storage/groups/2175303/settings' | json_pp
+```
+
+</TabItem>
+</Tabs>
+
+#### Responses
+
+<table id="table-api">
+<tbody>
+  <tr>
+    <td><code>200</code></td>
+    <td colSpan='2'>Success.</td>
+  </tr>
+</tbody>
+<tbody>
+  <tr>
+    <td><code>404</code></td>
+    <td colSpan='2'>Not found.</td>
+  </tr>
+</tbody>
+</table>
+
+```jsx title="Sample Response"
+{
+    "settings": {
```

Comment:
should we add a separate Android response here? mostly asking tech content how could we display that

Reply:
let's keep this open.. 
In the meantime, you could find out if there are only two different "sets of settings" android & ios.. 
if so.. we could do 2 separate tabs.. 
but if e.g. simulator etc have all different sets of settings.. that is not a good solution.

Reply:
^Other than the 'kind' and 'identifier' settings, are there different/additional settings for Android? If so, we could add tabs as @josipbudzaki-sl suggested.